lib/liveValidation/liveValidator.ts

```diff
@@ -834,15 +838,15 @@ export const parseValidationRequest = (
   let resourceType = "";
   let providerNamespace = "";
 
-  const parsedUrl = url.parse(requestUrl, true);
+  const parsedUrl = new URL(requestUrl, "https://management.azure.com");
   const pathStr = parsedUrl.pathname || "";
   if (pathStr !== "") {
     // Lower all the keys and values of query parameters before searching for `api-version`
-    const queryObject = _.transform(
-      parsedUrl.query,
-      (obj: ParsedUrlQuery, value, key) =>
-        (obj[key.toLowerCase()] = _.isString(value) ? value.toLowerCase() : value)
-    );
+    const queryObject: ParsedUrlQuery = {};
+    parsedUrl.searchParams.forEach((value, key) => {
+      queryObject[key.toLowerCase()] = value;
```

Comment:
Should value be lowercased when it's string?

Reply:
Key must be string

Reply:
I meant `value` that could be apiversion like 2020-01-01-PREview.

Reply:
Good catch